cli/src/pixl_cli/_io.py

```diff
@@ -67,6 +67,58 @@ def copy_parquet_return_logfile_fields(resources_path: Path) -> tuple[str, datet
     return project_name_slug, omop_es_timestamp
 
 
+def messages_from_csv(filepath: Path, omop_es_timestamp: datetime) -> list[Message]:
+    """
+    Reads patient information from CSV and transforms that into messages.
+    :param filepath: Path for CSV file to be read
+    """
+    expected_col_names = [
+        "procedure_id",
+        "mrn",
+        "accession_number",
+        "project_name",
+        "omop-es-datetime",
+    ]
+
+    # First line is column names
+    messages_df = pd.read_csv(filepath, header=0, dtype=str)
+
+    if list(messages_df.columns)[:5] != expected_col_names:
+        msg = f"csv file expected to have at least {expected_col_names} as " f"column names"
+        raise ValueError(msg)
+
+    (
+        procedure_id_col_name,
+        mrn_col_name,
+        acc_num_col_name,
+        project_col_name,
+        dt_col_name,
+    ) = expected_col_names
+
+    messages = []
+    for _, row in messages_df.iterrows():
+        message = Message(
+            mrn=row[mrn_col_name],
+            accession_number=row[acc_num_col_name],
+            study_date=datetime.strptime(row[dt_col_name], "%d/%m/%Y %H:%M")
+            .replace(tzinfo=timezone.utc)
+            .date(),
+            procedure_occurrence_id=row[
+                procedure_id_col_name
+            ],  # 4 row[4], #procedure_occurrence_id
+            project_name=row[project_col_name],
+            omop_es_timestamp=omop_es_timestamp,
```

Comment:
we should rename the Message field from `omop_es_timestamp` to something more generic like `extract_generated_timestamp`